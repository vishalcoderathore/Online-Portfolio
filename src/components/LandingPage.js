import React from 'react';

class LandingPage extends React.Component {
    
    onWelcome = () => {
        this.props.history.push('/dashboard');
    }

    render() {
        return (
            <div className="intro">
                <div className="content">
                    <h1 className="animated bounceInLeft">hello, hola, namaste</h1>
                </div>
                <div className="content animated bounceInRight">
                    <a className="btnLanding " onClick={this.onWelcome}>Welcome</a>
                </div>
            </div >
        );
    };
};

export default LandingPage;