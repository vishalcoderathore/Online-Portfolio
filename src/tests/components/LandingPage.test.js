import React from 'react';
import { shallow } from 'enzyme';
import LandingPage from '../../components/LandingPage';

test('should render Landing Page correctly', () => {
    const wrapper = shallow(<LandingPage />);
    expect(wrapper).toMatchSnapshot();
});

