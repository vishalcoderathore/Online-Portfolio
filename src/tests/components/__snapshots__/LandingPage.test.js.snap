// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Landing Page correctly 1`] = `
<div
  className="intro"
>
  <div
    className="content"
  >
    <h1>
      hello, hola, namaste
    </h1>
  </div>
  <div
    className="content"
  >
    <a
      className="btnLanding"
      onClick={[Function]}
    >
      Welcome
    </a>
  </div>
</div>
`;
